src/python/grpcio/grpc/_server.py

```diff
@@ -769,10 +869,28 @@ class _ServerStage(enum.Enum):
 
 
 class _ServerState(object):
+    lock: threading.RLock
+    completion_queue: cygrpc.CompletionQueue
+    server: cygrpc.Server
+    generic_handlers: Optional[List[grpc.GenericRpcHandler]]
+    interceptor_pipeline: Optional[_interceptor._ServicePipeline]
+    thread_pool: futures.ThreadPoolExecutor
+    stage: _ServerStage
+    termination_event: threading.Event
+    shutdown_events: Sequence
+    maximum_concurrent_rpcs: Optional[int]
+    active_rpc_count: int
+    rpc_states: Set[_RPCState]
+    due: Set[str]
+    server_deallocated: bool
 
     # pylint: disable=too-many-arguments
-    def __init__(self, completion_queue, server, generic_handlers,
-                 interceptor_pipeline, thread_pool, maximum_concurrent_rpcs):
+    def __init__(self, completion_queue: cygrpc.CompletionQueue,
+                 server: cygrpc.Server,
+                 generic_handlers: Optional[Sequence[grpc.GenericRpcHandler]],
```

Comment:
I don't think this can be `None` at this point.

Reply:
You're right, `Sequence` should be enough, will do the same for `interceptors` and `options`.